extra/docker/Dockerfile.centos7

```diff
@@ -0,0 +1,41 @@
+# Dockerfile for testing the build of gluster-block based on centos7
```

Comment:
Just fix up the comment since you made it more generic. Same with the other file.

Reply:
Sorry, it was unintentional. I have copied the template from gluster-block repo. Fixed it now.

Thanks!